server.js

```diff
@@ -0,0 +1,29 @@
+const server = require('express')();
+
+const dbRouter = require('./data/db-router.js');
+
+server.use(require('express').json());
+
+// Logger example from Express Middleware TK:
+server.use(logger);
+
+// Possible problem with syntax here:
+server.use(require('cors')());
```

Comment:
this should work but it would look cleaner when storing this import in a const `cors` at the top of the file, then you can just say `server.use(cors())`